open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/templates/egeria-core.yaml

```diff
@@ -15,8 +15,8 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: 8080
-      targetPort: 8080
+    - port: 9443
+      targetPort: 9443
       nodePort: {{ .Values.service.nodeport.core }}
```

Comment:
Logically we probably should update the node ports to be suitably numbers - it will still work without, but it seems more complete - ie rather than 30080 maybe it's 30043 (ports need to be 30000-32999 I think) etc